### ISSUE TITLE
Add an "Other actions" heading to details pages

### ISSUE THREAD

**Issue by outerpress** [department-of-veterans-affairs/va.gov-team#88859]
## Task Description

"Other actions" should include:
- Print button
- Cancel button
- Alert that shows when details pages can't be canceled.

## Notes and References
- CC ticket #88452
- This helps screenreader users who are navigating by heading discover the actions.

## Acceptance Criteria
- [ ] Design spec is complete

## Deliverables
- TBD

**Comment by Cieramaddox** [comment]
@outerpress I've added explorations to this ticket for you to review.
Next step (I believe): Take it to SME's to review language.

**Comment by benbrasso-agile6** [comment]
Added check-in labels and Epic to cross-track between scrums teams

**Comment by outerpress** [comment]
@cgednalske could you drop a link to the ticket for the unmoderated card sort here? I want to link this to that. Thanks!

**Comment by cgednalske** [comment]
@outerpress certainly! Here's the ticket for [creating the study](https://github.com/department-of-veterans-affairs/va.gov-team/issues/91617), as well as a link to the [research repo issue](https://github.com/department-of-veterans-affairs/va.gov-research-repository/issues/680).

**Comment by cgednalske** [comment]
@outerpress yes, definitely! Thanks for the reminder 🙏

**Comment by cgednalske** [comment]
@outerpress here are the findings and recommendations for Cancel and Print

#### Cancel and Reschedule

[Finding #4](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/appointments/va-online-scheduling/research/2024-08-appointment-details-card-sort/research-findings.md#finding-4-8-participants-created-cancelingrescheduling-categories-and-cancelingrescheduling-tasks-were-often-separate-from-other-tasksinformation): 8 Participants created canceling/rescheduling categories, and canceling/rescheduling tasks were often separate from other tasks/information

[Recommendation](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/appointments/va-online-scheduling/research/2024-08-appointment-details-card-sort/research-findings.md#recommendations-2): Separate/continue to separate this out from the Appointment info section into its own Cancel/Reschedule/Modify section. 

<img width="276" alt="Image" src="https://github.com/user-attachments/assets/f1a29948-16ab-494d-8bc3-162e0ce8eccd" />


#### Print

Participants associated print with Appointment info, but also most other categories. 

[Recommendation](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/appointments/va-online-scheduling/research/2024-08-appointment-details-card-sort/research-findings.md#recommendations-1): Because it is mentioned in many categories, pull the print button out of the appointment section so it’s on its own on the page.

<img width="264" alt="Image" src="https://github.com/user-attachments/assets/072209e1-346d-4812-b79c-1f506c743dbe" />

**Comment by outerpress** [comment]
Thanks @cgednalske ! I think we could approach this as two separate efforts. 
1. Decide where to put the "Manage appointment/request" section (actual name TBD)in details pages
2. Work with health apps team to find a consistent location for the print button across other spaces in those apps that have a similar "details" screen.

But we can talk more when we discuss as a team.
